### PR TITLE
Created a graph overview component

### LINKED ISSUE #13: Graph Overview
I've replicated Roam's Graph Overview for my own site and wondered if it was something you'd be interested in adding to the `gatsby-theme-brain` project or if it's something best left in user-land.

[Implementation is here](https://github.com/alexdunne/notes/blob/master/src/pages/graph-overview.js) if you're interested.

gatsby-theme-brain-graph-overview

### PR DESCRIPTION
**Pull request by alexdunne**
Graph overview similar to Roam's

I've followed the same approach used to create note pages to create the graph overview page.

Closes #13

### CODE REVIEW

**Comment by alexdunne** [comment]
I've committed the `yarn.lock` file but I noticed in another PR you added `package.lock` to `.gitignore` so I'm unsure about which you want to use on this project.

File: README.md [hunk 1]
```diff
@@ -84,4 +128,6 @@ If you want an example of how I shadow this, checkout my [websites github](https
 | `rootPath`               | "brain"                           | Set the root url for the brain on your site (e.g. in this case https://example.com/brain)                                                                        |
 | `rootNote`               | "brain"                           | Name of the 'index' note. So in this case brain.md would generate the root page of the brain                                                                     |
 | `generateSlug`           | `(filename) => slugify(filename)` | Function used to turn the filename of a note into its resulting slug (path)                                                                                      |
+| `graphOverviewTemplate`  | "./templates/graph-overview.js"   | Template to use for the graph overview                                                                                                                           |
+| `graphOverviewPath`      | "graph-overview"                  | The route for the graph overview (e.g. in this case https://example.com/brain)                                                                                   |
```
Comment (aengusmcmillin) [inline README.md]:
```suggestion
| `graphOverviewPath`      | "graph-overview"                  | The route for the graph overview (e.g. in this case https://example.com/graph-overview)                                                                 |
```

**Review by aengusmcmillin — APPROVED** [review]
One little change to the readme requested. And I think the yarn lockfile is fine. 

This looks great, thanks!

File: README.md (hunk 1 above)
Comment (alexdunne) [reply README.md]: Nice spot.

**Pull request merged**